soundbay/models.py

```diff
@@ -320,3 +320,47 @@ def forward(self, x):
             x = PCENTransform.pcen(x, self.eps, self.s, self.alpha, self.delta, self.r, self.training and self.trainable)
         x = x.unsqueeze(dim=1).permute((0, 1, 3, 2))
         return x
+
+
+class Squeezenet2D(nn.Module):
+    def __init__(self, num_classes=2, pretrained=True):
+        super(Squeezenet2D, self).__init__()
+
+        self.squeezenet = torch.hub.load('pytorch/vision:v0.10.0', 'squeezenet1_0',
+                                         pretrained=pretrained)
+        # number of features from existing squeezenet
+        num_features = 1000
```

Comment:
if it is possible, maybe it is better to read it from `self.squeezenet` / from the config